index.html

```diff
@@ -8,13 +8,19 @@
     <link rel="stylesheet" type="text/css" href="assets/styles/budgetkey-ng2-components.css">
     <script>
       window.prefetchedItem = {{ prefetchedItem | safe }};
+      function initTheme(){
+        var m, theme = (m = document.location.href.match('[?&]theme=([a-z]+)')) ? m[1] : "";
+        if (theme !== "") {
+          var script = document.createElement('script'); script.type = 'text/javascript'; script.src = "assets/theme."+theme+".js";
+          document.getElementsByTagName('head')[0].appendChild(script);
+        }
+      }initTheme();
```

Comment:
This entire block can be replaced with a self-called anonymous function:
```javascript
(function() {
  // place all from initBody() here
})();
```
This will prevent global scope from pollution with temporary functions/variables.

But, in my opinion, it's not so good to keep theme in js file. Can this be somehow replaced with json and loading it on app bootstrap? WDYT?

Reply:
IIRC we chose this method because we needed to do the theme loading before webpack - which complicates loading it as a json file

Reply:
fixed